src/cmd/builtin/fsck.c

```diff
@@ -64,21 +64,21 @@ static void fsck_treeobj (struct fsck_ctx *ctx,
 
 static void valref_validate (struct fsck_valref_data *fvd);
 
-static void read_verror (const char *fmt, va_list ap)
+static void output_verrmsg (const char *fmt, va_list ap)
```

Comment:
Suggestion: use a shorter name like `warn()`.  Repeating the long name all over the place gets tiresome.

Reply:
good idea, i could just go with "errmsg" :-)